sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -2201,6 +2246,97 @@ class RocksDBSuite extends AlsoTestWithChangelogCheckpointingEnabled with Shared
     }
   }
 
+  testWithChangelogCheckpointingEnabled("load the same version of pending snapshot uploading") {
```

Comment:
Is this the same with @siying just added? See the commit - https://github.com/apache/spark/commit/cf487b9f81fb8b95d0596111187b863dc9037855#diff-dc6f9dfe11e76f890ff2986f866853bcac263027c82562f9a52f4672a5460826R1860

Reply:
yep, thanks for catching this. Removed.